src/Eval.rsc

```diff
@@ -24,12 +22,92 @@ alias VEnv = map[str name, Value \value];
 data Input
   = input(str question, Value \value);
 
+Value getDefaultValue(AType typeName) {
+  switch (typeName) {
```

Comment:
Curly braces around cases

Reply:
A regex that should work for this is:

s/(\s*)case (.*):\n(.*)/\1case \2: {\n\3\1}/